SwiftSyntaxDevUtils/Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version:5.7
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+  name: "SwiftSyntaxDevUtils",
```

Comment:
We recently renamed the `swift-syntax` package to `swift-syntax`. For consistency, this should also need to be named `swift-syntax-dev-utils`.